apps/ejabberd/test/mongoose_rdbms_SUITE.erl

```diff
@@ -86,10 +85,10 @@ keepalive_exit(Config) ->
 meck_config(Server, KeepaliveInterval) ->
     meck:new(ejabberd_config, [no_link]),
     meck:expect(ejabberd_config, get_local_option,
-                fun({odbc_keepalive_interval, _Host}) -> KeepaliveInterval;
-                   ({odbc_start_interval, _Host}) -> 30;
+                fun({odbc_keepalive_interval, odbc_pool, default}) -> KeepaliveInterval;
```

Comment:
I know this is a test. It's also a code we read and this 3 element tuple is not intuitive at first glance. Would it be possible to mock `mongoose_rdbms_sup:get_option` here instead of `ejabberd_config:get_local_option`?

That's OK if you don't have time for this @chrzaszcz now, I'd just like to know that.